packages/@aws-cdk/toolkit/lib/actions/deploy/index.ts

```diff
@@ -173,6 +173,14 @@ export interface BaseDeployOptions {
    * @default 1
    */
   readonly concurrency?: number;
+
+  /**
+   * Whether to show logs from all CloudWatch log groups in the template
+   * locally in the users terminal
```

Comment:
Not sure what, but the Toolkit doesn't really have a concept of a user terminal. It works with the IoHost instead (which or might not print)

Reply:
good call out